draft-ietf-teep-architecture.md

```diff
@@ -172,7 +172,7 @@ The following terms are used:
 
   - Device: A physical piece of hardware that hosts one or more TEEs,
     often along with
-    a Rich Execution Environment. A device contains a default list
+    a REE. A device contains a default list
```

Comment:
This change is incorrect in my view, since the acronym REE is not defined until later (line 213).  Don't use acronyms before they're defined/expanded.

Reply:
Well. Previously I introduced the REE term in the introduction and hence this would not be the first occurrence. I added REE again into the introduction.